sdk/react/discoveryPanel.js

```diff
@@ -200,6 +222,40 @@ var DiscoveryPanel = React.createClass({
         </Text>
       </View>);
   },
+
+  renderError: function() {
+    var errorTitleText = "SOMETHING NOT RIGHT! THERE SHOULD BE VIDEOS HERE.";
+    var errorContentText = "(Try Clicking The Discover Button Again On Reload Your Page)";
+    var errorFlexDirectionStyle = {flexDirection: "row"};
+
+    if (this.props.width < widthThreshold) {
+      errorFlexDirectionStyle = {flexDirection: "column"};
+    }
+
+    var errorTitle = Utils.localizedString(this.props.locale, errorTitleText, this.props.localizedString);
+    var errorContent = Utils.localizedString(this.props.locale, errorContentText, this.props.localizedString);
+    var warningIcon = this.props.config.icons.warning.fontString;
```

Comment:
could you check for the warning icon first, please? because right now, until we update the skin config to a newer version, we'll get an error because the warning icon doesn't exist in our current branch of skin config.